samples/excel/16-custom-functions/custom-functions-errors.yaml

```
@@ -0,0 +1,37 @@
+order: 5
+id: excel-custom-functions-errors
+name: Custom functions errors
+description: Returns the "#NUM!" error as part of a 2-dimensional array.
+host: EXCEL
+api_set:
+    CustomFunctionsRuntime: 1.2
+script:
+    content: |-
+        /**
+        * Returns the #NUM! error as part of a 2-dimensional array.
+        * @customfunction
+        * @param {number} first First parameter.
+        * @param {number} second Second parameter.
+        * @param {number} third Third parameter.
+        * @returns {number[][]} Three results, as a 2-dimensional array.
+        */
+        function returnInvalidNumberError(first, second, third) {
+          // Use the `CustomFunctions.Error` object to retrieve an invalid number error.
```

Consider putting the function description inside so it shows up in the docs

```suggestion
        function returnInvalidNumberError(first, second, third) {
          /**
           * Returns the #NUM! error as part of a 2-dimensional array.
           * @customfunction
           * @param {number} first First parameter.
           * @param {number} second Second parameter.
           * @param {number} third Third parameter.
           * @returns {number[][]} Three results, as a 2-dimensional array.
           */

          // Use the `CustomFunctions.Error` object to retrieve an invalid number error.
```

---

The description *has* to be before the method. It uses JSDoc tags that are used to build the JSON metadata file. The JSDoc tags also work with IntelliSense. Since I'm going to add this snippet manually to the ref docs, I'll make sure to retain the preceding description.